### PR TITLE
Choreographer v2.0.1

### PR DESCRIPTION
**Pull request by szkris**
Moved Choreographer dtos respectively into dto.shared and dto.internal.

### CODE REVIEW

File: choreographer/src/main/java/eu/arrowhead/core/choreographer/database/service/ChoreographerDBService.java [hunk 1]
```diff
@@ -10,17 +10,12 @@
...
 import eu.arrowhead.common.database.entity.ServiceDefinition;
-import eu.arrowhead.common.database.repository.ChoreographerActionActionStepConnectionRepository;
-import eu.arrowhead.common.database.repository.ChoreographerActionPlanActionConnectionRepository;
-import eu.arrowhead.common.database.repository.ChoreographerActionRepository;
-import eu.arrowhead.common.database.repository.ChoreographerActionStepRepository;
-import eu.arrowhead.common.database.repository.ChoreographerActionStepServiceDefinitionConnectionRepository;
-import eu.arrowhead.common.database.repository.ChoreographerNextActionStepRepository;
-import eu.arrowhead.common.database.repository.ServiceDefinitionRepository;
-import eu.arrowhead.common.database.repository.ChoreographerActionPlanRepository;
+import eu.arrowhead.common.database.repository.*;
```
Comment (borditamas) [inline choreographer/src/main/java/eu/arrowhead/core/choreographer/database/service/ChoreographerDBService.java]: I prefer to import only what is really necessary.

File: choreographer/src/test/java/eu/arrowhead/core/choreographer/database/service/ChoreographerDBServiceTest.java [hunk 2]
```diff
@@ -3,11 +3,9 @@
 import eu.arrowhead.common.Utilities;
 import eu.arrowhead.common.database.entity.ChoreographerActionPlan;
 import eu.arrowhead.common.database.entity.ChoreographerActionStep;
-import eu.arrowhead.common.database.entity.ServiceDefinition;
 import eu.arrowhead.common.database.repository.*;
```
Comment (borditamas) [inline choreographer/src/test/java/eu/arrowhead/core/choreographer/database/service/ChoreographerDBServiceTest.java]: I prefer to import only what is really necessary.

File: core-common/src/main/java/eu/arrowhead/common/dto/internal/DTOConverter.java [hunk 3]
```diff
@@ -9,6 +9,8 @@
 import java.util.Map.Entry;
 import java.util.Set;
 
+import eu.arrowhead.common.database.entity.*;
```
Comment (borditamas) [inline core-common/src/main/java/eu/arrowhead/common/dto/internal/DTOConverter.java]: I prefer to import only what is really necessary.

**Review by borditamas — APPROVED** [review]
I had one remark (same thing in more class). Lets see what is the other's opinion about. However it's approved from my side.

**Review by rbocsi — APPROVED** [review]
I agree with Tamás.

**Pull request merged**